Arm/Hand/hand.hpp

```diff
@@ -27,52 +27,101 @@ class Hand
 
   Hand(sjsu::arm::WristJoint wrist) : wrist_(wrist) {}
```

Comment:
```suggestion
  Hand(sjsu::arm::WristJoint & wrist) : wrist_(wrist) {}
```
Try this fix and it should fix errors

Reply:
Hi, I tried to compile the code with your fix but it keeps giving me this error: 

` error: uninitialized reference member in 'class sjsu::RmdX&' [-fpermissive]
   28 | Hand(sjsu::arm::WristJoint & wrist) : wrist_(wrist) {}
      |   ^~~~`

Which reference member is the error referring to?

Reply:
Nevermind. I think I solved the error.

Reply:
sorry didn't see this earlier